README.md

```diff
@@ -12,30 +12,58 @@ npm install eslint-config-brightspace
 
 ## Usage
 
-Set the `extends` property in the `.eslintrc.json` file, replacing `<environment>` with the desired environment-specific config.
+Shared configurations can be directly exported or included as part of a custom configuration in the `eslint.config..js` file.
+```js
+export { nodeConfig as default } from './index.js';
+```
+
+```js
+import { nodeConfig } from 'eslint-config-brightspace';
+
+export default [
+	...nodeConfig,
+	// Custom configuration
+];
+```
+
```

Comment:
Should these two use cases maybe be split out to make it more clear what the options are? Also shouldn't the export be from `eslint-config-brightspace`? Something like:

Shared configurations can be directly exported from `eslint.config.js`:
```js
export { nodeConfig as default } from 'eslint-config-brightspace';
```

Or included as part of a custom configuration in `eslint.config..js`:
```js
import { nodeConfig } from 'eslint-config-brightspace';

export default [
	...nodeConfig,
	// Custom configuration
];
```